solidity/complex/interpreter/test.json

```diff
@@ -10,838 +10,1404 @@
+            "runtime_code": ""
+        },
         "SWAP11_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP11_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9A"
         },
 
+        "SWAP12_Before": {
+            "runtime_code": ""
+        },
         "SWAP12_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP12_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9B"
         },
 
+        "SWAP13_Before": {
+            "runtime_code": ""
+        },
         "SWAP13_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP13_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9C"
         },
 
+        "SWAP14_Before": {
+            "runtime_code": ""
+        },
         "SWAP14_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP14_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9D"
         },
 
+        "SWAP15_Before": {
+            "runtime_code": ""
+        },
         "SWAP15_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP15_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9E"
         },
 
+        "SWAP16_Before": {
+            "runtime_code": ""
+        },
         "SWAP16_Template": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F"
         },
         "SWAP16_Full": {
             "runtime_code" : "5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F5F9F"
         },
 
+        "LOG0_Before": {
+            "runtime_code": ""
+        },
+        "LOG0_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG0_Full": {
+            "runtime_code" : "602560005260206000A0"
+        },
+
+        "LOG1_Before": {
+            "runtime_code": ""
+        },
+        "LOG1_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG1_Full": {
+            "runtime_code" : "602560005260206000A1"
+        },
+
+        "LOG2_Before": {
+            "runtime_code": ""
+        },
+        "LOG2_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG2_Full": {
+            "runtime_code" : "602560005260206000A2"
+        },
+
+        "LOG3_Before": {
+            "runtime_code": ""
+        },
+        "LOG3_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG3_Full": {
+            "runtime_code" : "602560005260206000A3"
+        },
+
+        "LOG4_Before": {
+            "runtime_code": ""
+        },
+        "LOG4_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG4_Full": {
+            "runtime_code" : "602560005260206000A4"
+        },
+
+        "CREATE_Before": {
```

Comment:
Could you group these opcodes together?
Like calls, CREATEs, returns consequently.

Reply:
What do you mean here? Removing the spaces between them?

Reply:
No, grouping similar opcodes together.